assets/dev/js/editor/editor-base.js

```diff
@@ -14,6 +14,7 @@ import Navigator from './regions/navigator/navigator';
 import NoticeBar from './utils/notice-bar';
 import Preview from 'elementor-views/preview';
 import PopoverToggleControl from 'elementor-controls/popover-toggle';
+import ResponsiveBar from './regions/responsive-bar/responsive-bar';
```

Comment:
A. Is this an experiment?
B. Why not work with commands?
C. Screenshots before/after will be nice.

Reply:
@kobizz 

Any comments?

Reply:
A. No
B. Will do in the next phase
C. @joshmarom please attach here relevant screenshots

Reply:
![Screen Shot 2021-02-14 at 21 02 16](https://user-images.githubusercontent.com/2726148/107945170-e9e86b00-6f97-11eb-8c80-24d8543bffd3.png)
![Screen Shot 2021-02-14 at 21 03 23](https://user-images.githubusercontent.com/2726148/107945199-f5d42d00-6f97-11eb-80c6-4848b7e35d14.png)
![Screen Shot 2021-02-14 at 21 03 10](https://user-images.githubusercontent.com/2726148/107945219-fa004a80-6f97-11eb-91f1-8bc5f5d4d0f3.png)